sim/warlock/items.go

```diff
@@ -80,15 +80,15 @@ var ItemSetPlagueheartGarb = core.NewItemSet(core.ItemSet{
 				OnReset: func(aura *core.Aura, sim *core.Simulation) {
 					aura.Activate(sim)
 				},
-				OnCastComplete: func(aura *core.Aura, sim *core.Simulation, spell *core.Spell) {
```

Comment:
Seems like this change is a no-op? The new implementation in `life_tap.go` also applies the aura AFTER the calculation.

When reasonable, keeping the set bonus logic isolated to the set bonus function is preferred because its easier for the reader to understand.

Reply:
It is, yes. The only purpose of that commit was to prove that warlock DPS doesn't change from any other interaction.